Automattic/pocket-casts-ios
# Refactor Main Interface storyboard to SwiftUI

Fixes #https://github.com/Automattic/pocket-casts-ios/issues/23

Refactor Main Interface storyboard to SwiftUI.

| 1 | 2 |
| - | - |
| ![Simulator Screenshot - Apple Watch Series 9 (45mm) - 2024-03-19 at 09 19 12](https://github.com/Automattic/pocket-casts-ios/assets/651601/54285d79-351f-4b26-874f-92ab663e83bb) | ![Simulator Screenshot - Apple Watch Series 9 (45mm) - 2024-03-19 at 09 19 07](https://github.com/Automattic/pocket-casts-ios/assets/651601/6de6fecf-c993-484c-a2e9-3dc1fa59c143) | 


## To test

 - Select the scheme `Pocket Casts Watch App` start the app on a watch device
 - Start the app
 - Select the Watch source
 - Check the layout of the interface screen
 - Check if the counters for UpNext and Downloads rows are correct
 - Enter UpNext
 - Remove or add some elements to UpNext and see if the counter updates correctly in the main page
 - Enter UpNext or Podcast, select an episode and choose to download it
 - Go back to the interface screen and see if downloads counter updates correctly after download is finished
 - Repeat the above but go inside the Downloads screen and check if the new downloaded episode shows up when the download finish
 - Check if the PlayNow Row updates correctly when the the episode plays and it move to to the next one. ( on the simulator is not possible to play episode directly from the watch, only from the phone source)
 - Open Podcasts on the Watch source
 - Pull up list and check if sort options/toolbar shows correctly
 - Open files on the Watch source
 -  Pull up list and check if sort options/toolbar shows correctly

## Checklist
